.github/workflows/release_appimage.yml

```diff
@@ -46,13 +46,20 @@ jobs:
         mv cpeditor.desktop default.desktop
         mv cpeditor-icon.png icon.png
         ./linuxdeployqt*.AppImage ./cpeditor -appimage -qmake=/opt/qt512/bin/qmake
+    
+    - name: Package the complete Source
```

Comment:
Why not put this in the AUR workflow?

Reply:
Because this workflow already has the action for uploading to Github releases. Why duplicate it on AUR as well, AUR workflow will not use GitHub releases at all.

Reply:
It's not related to AppImage and doesn't require building.

Reply:
It is related with Linux and packing of source is not something that is optional. See build_test releases (Non-linux releases) DMG and .EXE and this release_appimage will release the Appimage and the source that was used to build it. It would be better to rename release_appimge to release linux. the release AUR workflow has only one thing that it uploads the AUR script to AUR website.

Reply:
release_aur should only release to AUR (by name). I will rename the release_appimage to release_linux, Also the source released by this could be used to create new workflows like `release_deb` or `release_rpm`. So it will be more generalized